src/NServiceBus.Core/Recoverability/Faults/MoveFaultsToErrorQueueBehavior.cs

```diff
@@ -71,12 +79,26 @@ public MoveFaultsToErrorQueueBehavior(CriticalError criticalError, IPipelineBase
         HostInformation hostInformation;
         BusNotifications notifications;
         string errorQueueAddress;
+        string localAddress;
         static ILog Logger = LogManager.GetLogger<MoveFaultsToErrorQueueBehavior>();
 
         public class Registration : RegisterStep
         {
-            public Registration()
-                : base("MoveFaultsToErrorQueue", typeof(MoveFaultsToErrorQueueBehavior), "Moved failing messages to the configured error queue")
+            public Registration(ReadOnlySettings settings, string localAddress)
+                : base("MoveFaultsToErrorQueue", typeof(MoveFaultsToErrorQueueBehavior), "Moved failing messages to the configured error queue", b =>
+                {
```

Comment:
I wonder if we even need the registration. I think they only add value when you need to specify the order (which we should be doing less and less)